packages/mui-system/src/style.js

```diff
@@ -100,4 +100,18 @@ function style(options) {
   return fn;
 }
 
+function isColor(str) {
```

Comment:
Not a fan of this implementation. Till we find a proper solution, I'd propose we remove the warning to not spam the console.